examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,249 @@ Documentation for other releases can be found at
-
-You will need to create the disks in the same [GCE zone](https://cloud.google.com/compute/docs/zones) as the Kubernetes cluster. The default setup script will create the cluster in the `us-central1-b` zone, as seen in the [config-default.sh](../../cluster/gce/config-default.sh) file. Replace `$ZONE` below with the appropriate zone.
-
-We will create two disks: one for the mysql pod, and one for the wordpress pod. In this example, we create 20GB disks, which will be sufficient for this demo. Feel free to change the size to align with your needs, as wordpress requirements can vary. Also, keep in mind that [disk performance scales with size](https://cloud.google.com/compute/docs/disks/#comparison_of_disk_types).
-
-First create the mysql disk.
-
-```sh
-gcloud compute disks create --size=20GB --zone=$ZONE mysql-disk
+## Table of Contents
+
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+
+- [Persistent Installation of MySQL and WordPress on Kubernetes](#persistent-installation-of-mysql-and-wordpress-on-kubernetes)
+  - [tl;dr Quickstart](#tldr-quickstart)
+  - [Table of Contents](#table-of-contents)
+  - [Cluster Requirements](#cluster-requirements)
+  - [Decide where you will store your data](#decide-where-you-will-store-your-data)
+    - [Host Path](#host-path)
+    - [GCE Persistent Disk](#gce-persistent-disk)
+  - [Create the MySQL Password Secret](#create-the-mysql-password-secret)
+  - [Deploy MySQL](#deploy-mysql)
+  - [Deploy WordPress](#deploy-wordpress)
+  - [Visit your new WordPress blog](#visit-your-new-wordpress-blog)
+  - [Take down and restart your blog](#take-down-and-restart-your-blog)
+  - [Next Steps](#next-steps)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+## Cluster Requirements
+
+Kubernetes runs in a variety of environments and is inherently
+modular. Not all clusters are the same. These are the requirements for
+this example.
+
+* Kubernetes version 1.2 is required due to using newer features, such
+  at PV Claims and Deployments. Run `kubectl version` to see your
+  cluster version.
+* [Cluster DNS](../../cluster/addons/dns/) will be used for service discovery.
+* An [external load balancer](http://kubernetes.io/docs/user-guide/services/#type-loadbalancer)
+  will be used to access WordPress.
+* [Persistent Volume Claims](http://kubernetes.io/docs/user-guide/persistent-volumes/)
+  are used. You must create Persistent Volumes in your cluster to be
+  claimed. This example demonstrates how to create two types of
+  volumes, but any volume is sufficient.
+
+Consult a
+[Getting Started Guide](http://kubernetes.io/docs/getting-started-guides/)
+to set up a cluster and the
+[kubectl](http://kubernetes.io/docs/user-guide/prereqs/) command-line client.
+
+## Decide where you will store your data
+
+MySQL and WordPress will each use a
```

Comment:
```
We will use a Persistent Volume Claim to claim an available persistent volume.
```

This reads awkwardly.  Maybe just embed this in the previous sentence to read 

MySQL and WordPress will each use a Persistent Volume with a Persistent Volume Claim to store their data.